import/dproto/serialize.d

```diff
@@ -312,6 +317,8 @@ unittest {
  * Params:
  *  	src = The data stream
  * Returns: The decoded value
+ * 
+ * Note: This is @trusted for compatibility with @system R.put().
```

Comment:
Seems potentially problematic to me to blindly trust the code for any arbitrary `R`.